ui/apps/platform/src/sagas/authSagas.js

```diff
@@ -282,7 +282,9 @@ function* saveAuthProvider(action) {
             yield call(fetchUsersAttributes);
             yield put(actions.selectAuthProvider({ ...remaining, id: savedAuthProvider.data.id }));
         } else {
-            yield call(AuthService.saveAuthProvider, remaining);
+            if (!remaining.active && !remaining.traits?.mutabilityMode === 'ALLOW_FORCED') {
```

Comment:
```suggestion
            if (!remaining.active && !(remaining.traits?.mutabilityMode === 'ALLOW_FORCED')) {
```

This currently leads to failing UI e2e tests ([see here](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/stackrox_stackrox/2610/pull-ci-stackrox-stackrox-master-gke-ui-e2e-tests/1557292002603700224)), which try to update an auth provider whilst not having set the `traits` property at all.